bot.js

```diff
@@ -0,0 +1,25 @@
+//Login to Discord
+const Discord = require('discord.js');
+const bot = new Discord.Client();
+const config = require('./config.json');
```

Comment:
The file extension is not necessary

```suggestion
const config = require('./config');
```